seqr/views/utils/pedigree_info_utils.py

```diff
@@ -411,11 +411,11 @@ def _set_proband_relationship(json_records):
 
         maternal_siblings = {
             r[JsonConstants.INDIVIDUAL_ID_COLUMN] for r in records
-            if affected[JsonConstants.MATERNAL_ID_COLUMN] == r[JsonConstants.MATERNAL_ID_COLUMN]
+            if affected[JsonConstants.MATERNAL_ID_COLUMN] and affected[JsonConstants.MATERNAL_ID_COLUMN] == r[JsonConstants.MATERNAL_ID_COLUMN]
         }
         paternal_siblings = {
             r[JsonConstants.INDIVIDUAL_ID_COLUMN] for r in records
-            if affected[JsonConstants.PATERNAL_ID_COLUMN] == r[JsonConstants.PATERNAL_ID_COLUMN]
+            if affected[JsonConstants.PATERNAL_ID_COLUMN] and affected[JsonConstants.PATERNAL_ID_COLUMN] == r[JsonConstants.PATERNAL_ID_COLUMN]
```

Comment:
I did fix a small bug I caught here when updating the test, which is that previously if 2 individuals were both missing one parent ID we were counting them as half siblings because they had the "same" parent (which is "None")